Extensions/old_blue.js

```diff
@@ -9,6 +10,13 @@ XKit.extensions.old_blue = new Object({
 
 	running: false,
 
+	preferences: {
+		"note": {
+			text: `<span style="text-transform:none"><a href="https://raw.githubusercontent.com/new-xkit/XKit/master/Extensions/old_blue.css" target="_blank">CSS file</a> for usage in userstyle add-ons</span>`,
```

Comment:
not sure how I feel about the wording here, could stand to be a little more comprehensible

Reply:
open to suggestions, will give it another go myself tomorrow